gaffer-as-a-service/gaas-rest/pom.xml

```diff
@@ -295,5 +296,63 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>docker-build</id>
```

Comment:
Is the objective just to output the docker file? If that's the case then you could remove the profile and have it as a build plugin instead.